UKHomeOffice/cerberus-service
# COP-12016: RBAC for Targeter UI

## Description
Added role base access for targets

## To Test
Please see the rules here:
https://support.cop.homeoffice.gov.uk/browse/COP-12016

## Developer Checklist

\* Required

- [*] Up-to-date with main branch? \*

- [*] Does it have tests?

- [*] Pull request URL linked to JIRA ticket? \*

- [*] All reviewer comments resolved/answered? \*
